### PR TITLE
Release/56.0.0

### PR DESCRIPTION
**Pull request by MajorLift**
This is the release candidate for v56.0.0.

- `@metamask/snaps-rpc-methods`: major
- `@metamask/snaps-sdk`: minor
- `@metamask/snaps-utils`: minor

### CODE REVIEW

File: packages/snaps-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- **BREAKING:** Add non-restricted RPC method `snap_resolveInterface`, which allows a snap to resolve a given user interface bound to a `snap_dialog` with a custom value. ([#2509](https://github.com/metamask/snaps/pull/2509))
+  - Add new types `ResolveInterfaceParams`, `ResolveInterfaceResult`.
+- feat: Card component ([#2480](https://github.com/metamask/snaps/pull/2480))
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:13]:
```suggestion
- Add `Card` component ([#2480](https://github.com/metamask/snaps/pull/2480))
```

File: packages/snaps-sdk/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- **BREAKING:** Add non-restricted RPC method `snap_resolveInterface`, which allows a snap to resolve a given user interface bound to a `snap_dialog` with a custom value. ([#2509](https://github.com/metamask/snaps/pull/2509))
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:11]: While this PR is breaking, it doesn't break `snaps-sdk`. We may want to do a minor bump here.

```suggestion
- Add non-restricted RPC method `snap_resolveInterface` ([#2509](https://github.com/metamask/snaps/pull/2509))
   - This method allows a Snap to resolve a given user interface bound to a `snap_dialog` with a custom value.
```

File: packages/snaps-sdk/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - Add new types `ResolveInterfaceParams`, `ResolveInterfaceResult`.
+- feat: Card component ([#2480](https://github.com/metamask/snaps/pull/2480))
+- Add `BoxChildStruct`, `FormChildStruct`, `FieldChildUnionStruct`. ([#2409](https://github.com/metamask/snaps/pull/2409))
+- Add `Container` and `Footer` components ([#2517](https://github.com/metamask/snaps/pull/2517))
+
+### Changed
+- `RootJSXElementStruct` is updated so that the allowed JSX elements at the root are the null union of the Container component and the allowed children of the Box component. ([#2526](https://github.com/metamask/snaps/pull/2526))
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:18]:
```suggestion
- Update `RootJSXElement` to allow `Container` or `Box` at the root ([#2526](https://github.com/metamask/snaps/pull/2526))
```

File: packages/snaps-utils/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.8.0]
+### Added
+- Add function `serialiseJsx`, which serializes a JSX node to a string. ([#2409](https://github.com/metamask/snaps/pull/2409))
```
Comment (Mrtenz) [inline packages/snaps-utils/CHANGELOG.md:11]: This function was moved. Maybe something like this is better?
```suggestion
- Move `serialiseJsx` function from `snaps-jest` to `snaps-utils` ([#2409](https://github.com/metamask/snaps/pull/2409))
```

File: packages/snaps-utils/CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Bump `@metamask/key-tree` from `^9.1.1` to `^9.1.2` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/permission-controller` from `^10.0.0` to `^10.0.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/snaps-registry` from `^3.1.0` to `^3.2.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+
+### Fixed
+- Replace `superstruct` with ESM-compatible `@metamask/superstruct` `^3.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+  - This fixes the issue of this package being unusable by any TypeScript project that uses `Node16` or `NodeNext` as its `moduleResolution` option.
+- Prevent `allowedOrigins` from being bypassed by enforcing that regex are terminated with $. ([#2576](https://github.com/metamask/snaps/pull/2576))
```
Comment (Mrtenz) [inline packages/snaps-utils/CHANGELOG.md:24]:
```suggestion
- Fix `allowedOrigins` bypass caused by unterminated regex ([#2576](https://github.com/metamask/snaps/pull/2576))
```

File: packages/snaps-sdk/CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Bump `@metamask/key-tree` from `^9.1.1` to `^9.1.2` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+
+### Fixed
+- Add `typedUnion` utility which strictly validates members of the union based on a type field. ([#2534](https://github.com/metamask/snaps/pull/2534))
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:24]: This is an internal function, so it doesn't need to be mentioned in the changelog.

```suggestion
```

File: packages/snaps-rpc-methods/CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook which is meant to bind to the `addAndShowRequest` method of the `ApprovalController`. ([#2509](https://github.com/metamask/snaps/pull/2509))
```
Comment (Mrtenz) [inline packages/snaps-rpc-methods/CHANGELOG.md:11]:
```suggestion
- **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook ([#2509](https://github.com/metamask/snaps/pull/2509))
   - It should bind to the `addAndShowRequest` method of the `ApprovalController`.
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: packages/snaps-sdk/CHANGELOG.md (hunk 2 above)
Comment (MajorLift) [reply packages/snaps-sdk/CHANGELOG.md]: Downgraded `snaps-sdk` to minor release: https://github.com/MetaMask/snaps/pull/2581/commits/79e11a92113b1e4860cbe0bf73cb75362a7ff08e

File: packages/snaps-rpc-methods/CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook ([#2509](https://github.com/metamask/snaps/pull/2509))
+   - It should bind to the `addAndShowRequest` method of the `ApprovalController`.
```
Comment (Mrtenz) [inline packages/snaps-rpc-methods/CHANGELOG.md:12]: Made a small mistake in my original suggestion. 😅
```suggestion
  - It should bind to the `addAndShowRequest` method of the `ApprovalController`.
```

File: packages/snaps-sdk/CHANGELOG.md [hunk 9]
```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
+### Added
+- Add non-restricted RPC method `snap_resolveInterface` ([#2509](https://github.com/metamask/snaps/pull/2509))
+   - This method allows a Snap to resolve a given user interface bound to a `snap_dialog` with a custom value.
+  - Add new types `ResolveInterfaceParams`, `ResolveInterfaceResult`.
+- Add `Card` component ([#2480](https://github.com/metamask/snaps/pull/2480))
+- Add `BoxChildStruct`, `FormChildStruct`, `FieldChildUnionStruct`. ([#2409](https://github.com/metamask/snaps/pull/2409))
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:15]: One more:
```suggestion
- Add `BoxChildStruct`, `FormChildStruct`, `FieldChildUnionStruct` ([#2409](https://github.com/metamask/snaps/pull/2409))
```
We generally don't end the "main" changelog entry with a period.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**